homework2/task3.c

```diff
@@ -0,0 +1,207 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <stdbool.h>
+
+#define arrElementType int
+
+// #define DEBUG 1
+
+void swap(arrElementType *a, arrElementType *b)
+{
+    arrElementType tmp = *a;
+    *a = *b;
+    *b = tmp;
+}
+
+void bubbleSort(size_t len, arrElementType *arr)
+{
+    for (size_t i = 0; i < len; ++i)
```

Comment:
```suggestion
    for (size_t i = 0; i < len - 1; ++i)
```
Последняя итерация избыточна